datasrc/fonts/index.json

```diff
@@ -0,0 +1,16 @@
+{
+    "font files": [
+        "DejaVuSans.ttf",
+        "SourceHanSans.ttc"
+    ],
+    "default": "DejaVu Sans",
```

Comment:
This value seems to be ignored. Instead it seems to always take the first font in `font files`